lib/rubocop/cop/layout/space_around_keyword.rb

```diff
@@ -212,10 +217,18 @@ def accept_left_square_bracket?(range)
           ACCEPT_LEFT_SQUARE_BRACKET.include?(range.source)
         end
 
+        def accept_namesapce_operator?(range)
```

Comment:
Typo: `sapce`

Reply:
Fixed. I already squashed and force pushed it; please let me know if the preference is to use separate commits and squash after approval.